src/nginxconfig/util/import_data.js

```diff
@@ -112,6 +113,9 @@ export default (query, domains, global, nextTick) => new Promise(resolve => {
         domains.push(clone(Domain.delegated));
     }
 
+    // Handle converting vue params to the current version
+    vueBackwardsCompatibility(data, domains);
```

Comment:
I would prefer this be applied directly after the angular compatibility layer, using only `data`, before we start pushing stuff into the Vue data structures.